lib/backend/api/api.go

```diff
@@ -105,6 +105,10 @@ type Client interface {
 	// EnsureInitialized ensures that the backend is initialized
 	// any ready to be used.
 	EnsureInitialized() error
+
+	// EnsureNodeInitialized ensures that the backend is initialized
```

Comment:
Would be good to go into detail w.r.t when this should be called.  Before / after creating the node resource?